File: account-service/src/main/java/finos/traderx/accountservice/model/Person.java
package finos.traderx.accountservice.model;

public class Person {
  private String logonId;
  private String fullName;
  private String email;
  private String department;
  private String photoUrl;

  public String getLogonId() {
    return logonId;
  }

  public void setLogonId(String logonId) {
    this.logonId = logonId;
  }

  public String getFullName() {
    return fullName;
  }

  public void setFullName(String fullName) {
    this.fullName = fullName;
  }

  public String getEmail() {
    return email;
  }

  public void setEmail(String email) {
    this.email = email;
  }

  public String getDepartment() {
    return department;
  }

  public void setDepartment(String department) {
    this.department = department;
  }

  public String getPhotoUrl() {
    return photoUrl;
  }

  public void setPhotoUrl(String photoUrl) {
    this.photoUrl = photoUrl;
  }
}

File: templates/people-service-specfirst/PeopleService.WebApi/Directory/Person.cs
namespace PeopleService.WebApi.Directory;

public sealed class Person
{
    public string LogonId { get; init; } = "";
    public string FullName { get; init; } = "";
    public string Email { get; init; } = "";
    public string EmployeeId { get; init; } = "";
    public string Department { get; init; } = "";
    public string PhotoUrl { get; init; } = "";
}
